db/schema.rb

```diff
@@ -44,6 +54,7 @@
     t.string   "last_sign_in_ip"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "name"
```

Comment:
Any thoughts on keeping a first and last name instead? If users are signing in via SSO (facebook, google), those already make the distinction between first and last so it wouldn't be any extra work to find it out.